ext/info.html

```diff
@@ -50,6 +50,7 @@ <h2 id="links">Links</h2>
                 <li>Support and source code: <a href="https://github.com/themoeway/yomitan" rel="noreferrer noopener">Github</a></li>
                 <li>Release notes: <a href="https://github.com/themoeway/yomitan/releases" rel="noreferrer noopener" data-href-format="https://github.com/themoeway/yomitan/releases/tag/{version}" id="release-notes-this-version-link">This version</a> | <a href="https://github.com/themoeway/yomitan/releases" rel="noreferrer noopener">All versions</a></li>
                 <li>More extension information: <a href="/permissions.html">Permissions</a> | <a href="/legal.html">Licenses</a> | <a href="/issues.html">Issues</a></li>
+                <li>Enjoying Yomitan? Help us by writing a review on the Firefox/Chrome store!<a href="https://addons.mozilla.org/en-US/firefox/addon/yomitan/">Firefox</a> | <a href="https://chromewebstore.google.com/detail/yomitan/likgccmbimhjbgkjambclfkhldnlhbnn">Chrome</a></li>
```

Comment:
There needs to be a space after `Chrome store!`.

```suggestion
                <li>Enjoying Yomitan? Help us by writing a review on the Firefox/Chrome store! <a href="https://addons.mozilla.org/en-US/firefox/addon/yomitan/">Firefox</a> | <a href="https://chromewebstore.google.com/detail/yomitan/likgccmbimhjbgkjambclfkhldnlhbnn">Chrome</a></li>
```